x-pack/plugins/cases/public/components/cases_context/index.tsx

```diff
@@ -148,8 +145,8 @@ export const CasesProvider: React.FC<{ value: CasesContextProps }> = ({
     [getFilesClient, owner]
   );
 
-  return isCasesContextValue(value) ? (
```

Comment:
I think I might have asked this in a previous PR, so sorry in advance 😅 

What about `permissions` being null?

I am assuming that is handled down the line by the components that need them. Is that correct?

Reply:
Yes, this is passed by the components that use our context provider. They should not be `null` ever.